domain/schema/schema_test.go

```diff
@@ -346,6 +350,8 @@ func (s *schemaSuite) TestControllerChangeLogTriggers(c *gc.C) {
 		"trg_log_secret_backend_rotation_next_rotation_time_insert",
 		"trg_log_secret_backend_rotation_next_rotation_time_update",
 		"trg_log_secret_backend_rotation_next_rotation_time_delete",
+		"trg_secret_backend_immutable_backends_update",
```

Comment:
We should probably change the name of this test function, as these are not change log triggers.

Reply:
Yeah, makes sense. Splitting it to a new test.